mmpose/datasets/datasets/body3d/body3d_base_dataset.py

```diff
@@ -60,6 +60,9 @@ def load_config(self, data_cfg):
         self.seq_len = data_cfg.get('seq_len', 1)
         self.seq_frame_interval = data_cfg.get('seq_frame_interval', 1)
         self.causal = data_cfg.get('causal', True)
+        self.pad = data_cfg.get('pad', False)
+        self.subset = data_cfg.get('subset', 1)
+        self.need_target_2d = data_cfg.get('need_target_2d', False)
```

Comment:
What does `need_target_2d` mean?

Reply:
It is used for unlabeled data. we first project the predicted 3D pose  back to 2D image plane, and then calculate the difference from target_2d. Since the input is a sequence of 2D poses, we need to specify the target frame.

Reply:
Consider using "need_2d_label" or "need_2d_groundtruth"  which may be more clear as a config option?

Reply:
OK. I think "need_2d_label" is better.